lib/ffeature/feature.rb

```diff
@@ -7,11 +7,14 @@ def initialize(feature)
       @feature = feature
     end
 
-    def enabled?(user)
+    def enabled?(user = nil)
       return true if FFeature.dev_mode
-      return false unless flipper_user?(user)
 
-      FFeature.flipper[feature].enabled?(user.to_model)
+      if flipper_user?(user)
+        FFeature.flipper[feature].enabled?(user.to_model)
+      else
+        FFeature.flipper[feature].enabled?
+      end
```

Comment:
Too late, my friend!

But in this particular case
1. if-else clause shows the difference between checking feature state based on user or not
2. return based is too long to read